index.js

```diff
@@ -111,15 +112,56 @@ function processMixinContent(rule, from) {
   })
 }
 
-function insertObject(rule, obj) {
+function insertObject(rule, obj, singeArgumentsMap) {
   let root = parse(obj)
   root.each(node => {
     node.source = rule.source
   })
   processMixinContent(root, rule)
+  unwrapSingleArguments(root.nodes, singeArgumentsMap)
   rule.parent.insertBefore(rule, root)
 }
 
+function unwrapSingleArguments(rules, singleArgumentsMap) {
+  if (singleArgumentsMap.size <= 0) {
+    return
+  }
+
+  for (let rule of rules) {
+    switch (rule.type) {
+      case 'decl':
+        if (rule.value.includes(SINGLE_ARGUMENT_KEYWORD)) {
+          let newValue = rule.value
+          for (let [key, value] of singleArgumentsMap) {
+            newValue = newValue.replace(key, value)
+          }
+          rule.value = newValue
+        }
+        break
+      case 'rule':
+        unwrapSingleArguments(rule.nodes, singleArgumentsMap)
+        break
+      default:
+        break
+    }
+  }
+}
+
+function resolveSingleArgumentValue(value) {
+  let content = value.slice(SINGLE_ARGUMENT_KEYWORD.length).trim()
+
+  if (!content.startsWith('(') || !content.endsWith(')')) {
+    throw new Error(
```

Comment:
This error will not be connected with CSS line and column.

Use `Node#error` instead.